### ISSUE TITLE
Error After Clicking Refresh Icon

### ISSUE THREAD

**Issue by clarkio** [Microsoft/vscode-pull-request-github#501]
- Extension version: 0.0.19
- VSCode Version: 1.28.0-insider (system setup)
- OS: Windows 10

Steps to Reproduce:

1. Install the latest Insiders build (d59d9f5aa5708c006e409c1a7b197c27fecb54a4)
2. Open a project that uses GitHub for source control
3. Click the "refresh" icon

You should see the following error:
![image](https://user-images.githubusercontent.com/6265396/45711823-bef45580-bb58-11e8-8d28-5a96cb2ab24e.png)

**Comment by clarkio** [comment]
Ok, thanks for the response @RMacfarlane. I'm trying what you suggested now and will report back.

**Comment by clarkio** [comment]
Seems to be working better after doing what you suggested but I had to uninstall system level install for both stable and insiders then install user level for both. The stable build works fine with the extension still and the insiders is working better but noticed a duplicate refresh icon:
![image](https://user-images.githubusercontent.com/6265396/45771582-7b125680-bc13-11e8-9f30-f91669db2bb5.png)

**Comment by RMacfarlane** [comment]
Ah, sorry, I was unclear - I just mean you had to uninstall and reinstall the extension, not VSCode itself. Did you do that in addition to the VSCode install?

**Comment by clarkio** [comment]
My bad you were definitely clear -- I wasn't in my response. I did the extension uninstall, reinstall, reload VS Code and I was still running into the issue with the extension (for Insiders). So I went a step further by trying to change to a new install of VS Code and I believe that's what helped solve it.

I think the issue may be a combination of the editor version and extension version. Just want to share the outcome in case it helps root out some weird edge case.

Thanks for your help and responsiveness 🙂
